src/status_im/contexts/communities/sharing/events.cljs

```diff
@@ -70,6 +67,24 @@
                                   :chat-id chat-id
                                   :event   :communities/get-community-channel-share-data}))}]})))
 
+(rf/reg-event-fx :communities/share-community-url-with-data
+ (fn [_ [community-id]]
+   (let [title      (i18n/label :t/community-on-status)
+         on-success (fn [url]
+                      (rf/dispatch [:open-share
+                                    {:options (if platform/ios?
+                                                {:activityItemSources
+                                                 [{:placeholderItem {:type    :text
+                                                                     :content title}
+                                                   :item            {:default {:type    :url
+                                                                               :content url}}
+                                                   :linkMetadata    {:title title}}]}
+                                                {:title     title
+                                                 :subject   title
+                                                 :url       url
+                                                 :isNewTask true})}]))]
+     {:fx [[:dispatch [:communities/get-community-share-data community-id on-success]]]})))
+
```

Comment:
Here we're adding the definition for `:communities/share-community-url-with-data` because it was accidentally removed.